### ISSUE TITLE
Add Filter to search items

### ISSUE THREAD

**Issue by rizkyramadhan** [OneGraph/graphiql-explorer#30]
Currently there are no filters, this is make searching for available query harder if there are hundred of it....

**Comment by FluorescentHallucinogen** [comment]
@rizkyramadhan Totally agree. PTAL at #28. See also video teaser: https://youtu.be/QFbTmyJFr4g?t=4m02s. :wink:

**Comment by rizkyramadhan** [comment]
Wow!, can I try that right now ?

**Comment by rizkyramadhan** [comment]
:heart_eyes:

**Comment by Nohac** [comment]
@FluorescentHallucinogen How is this feature coming along? This would be really useful to have!

**Comment by FluorescentHallucinogen** [comment]
@Nohac Totally agree. This is a very useful feature. But I'm only the author of the idea, not implementation. Only @sah and @sgrove can help.

**Comment by Toub** [comment]
A simple explorer search/filter would be awesome!

Looks like an easy to implement but very useful feature. However, is this project still maintained?

**Comment by sgrove** [comment]
@Toub Yup, maintained, and with lots of features we haven't even written about now.

Also, the feature is surprisingly intricate to implement properly.

**Comment by Toub** [comment]
Great to hear.
Thanks for the feedback.
